features/channel/managing_channels/adding_channel.feature

```diff
@@ -6,12 +6,14 @@ Feature: Adding a new channel
 
     Background:
         Given I am logged in as an administrator
+        And the store has currency "Euro"
```

Comment:
```
Given the store has currency "Euro"
And I am logged in as an administrator
```

At the beginning we should configure shop and login after everything is setup (we follow this convention everywhere).